docs/src/pages/customization/spacing/spacing.md

```diff
@@ -48,8 +47,14 @@ theme.spacing(2); // = 8
 ## Multiple arity
 
 The `theme.spacing()` helper accepts up to 4 arguments.
-You can use the arguments to reduce the boilerplate:
-```diff
--  padding: `${theme.spacing(1)}px ${theme.spacing(2)}px`,
-+  padding: theme.spacing(1, 2), // '8px 16px'
+You can use the arguments to reduce the boilerplate. Instead of doing:
```

Comment:
What's the problem with the diff approach?

Reply:
I remember hearing people complaining about it, that it would be more clear, without, with a better syntax coloration.